easybuild/framework/easyconfig/format/format.py

```diff
@@ -54,7 +54,7 @@
 
 # values for these keys will not be templated in dump()
 EXCLUDED_KEYS_REPLACE_TEMPLATES = ['description', 'easyblock', 'homepage', 'name', 'toolchain', 'version'] \
-                                  + DEPENDENCY_PARAMETERS
+                                  + ['exts_list'] + DEPENDENCY_PARAMETERS
```

Comment:
@ocaisa Let's keep this sorted alphabetically, it'll be way easier to tell whether a particular parameter is included or not that way upon visual inspection...

Done in https://github.com/ocaisa/easybuild-framework/pull/38